content/blog/2024-01-05-cooperative/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cooperative
+author: Yanina Bellini Saibene
+summary: ""
+date: '2024-01-05'
+tags:
+  - English
+  - Community
+---
+
+{{< figure src="featured.jpg" alt="Cooperative flag. The flag consisted of the colors red, orange, green, yellow, blue, indigo and violet and symbolized the ideals and objectives of universal peace, unity that overcomes political, economic, social, racial or religious differences and the hope of humanity in a better world where freedom, personal dignity, social justice and solidarity reign.">}}
+> The flag consisted of the colors red, orange, green, yellow, blue, indigo and violet and symbolized the ideals and objectives of universal peace, unity that overcomes political, economic, social, racial or religious differences and the hope of humanity in a better world where freedom, personal dignity, social justice and solidarity reign.
+
```

Comment:
```suggestion
My last few years being part of international non-profit organizations and communities of practice has made me reflect on the types of governance and business models that are mostly used and how successful models are ignored in these latitudes.  This article is about the cooperative model. A way of doing things that has been present in my life since I was a child.   
```